GW2EIEvtcParser/EncounterLogic/Raids/W8/GreerTheBlightbringer.cs

```diff
@@ -90,6 +116,35 @@ protected override List<TrashID> GetTrashMobsIDs()
         ];
     }
 
+    internal override void EIEvtcParse(ulong gw2Build, EvtcVersionEvent evtcVersion, FightData fightData, AgentData agentData, List<CombatItem> combatData, IReadOnlyDictionary<uint, ExtensionHandler> extensions)
+    {
+        base.EIEvtcParse(gw2Build, evtcVersion, fightData, agentData, combatData, extensions);
+
+        // Renaming Greer for CM
+        SingleActor? greer = Targets.FirstOrDefault(x => x.IsSpecies(TargetID.Greer));
```

Comment:
replace with 
SingleActor greer = Targets.FirstOrDefault(x => x.IsSpecies(TargetID.Greer)) ?? throw new MissingKeyActorsException("Greer not found");

The absence of Greer should throw an exception